test/test_quantized_tensor.py

```diff
@@ -172,10 +174,40 @@ def test_qtensor_permute(self):
         # compare dequantized result
         self.assertTrue(np.array_equal(qr1.dequantize().numpy(), qr2.dequantize().numpy()))
         # compare permuted + dequantized result with original transposed result
-        self.assertTrue(np.allclose(qr2.dequantize().numpy(), r.numpy().T, atol=2 / scale))
+        self.assertTrue(np.allclose(qr2.dequantize().numpy(), r.numpy().transpose([1, 0, 2, 3]), atol=2 / scale))
         # make permuted result contiguous
         self.assertTrue(torch.equal(qr2.contiguous().int_repr(), qr2.int_repr()))
 
+        # change memory format
+        qlast = qr.contiguous(memory_format=torch.channels_last)
+        self.assertEqual(qr.stride(), list(reversed(sorted(qr.stride()))))
+        self.assertNotEqual(qlast.stride(), list(reversed(sorted(qlast.stride()))))
+        self.assertTrue(torch.equal(qr.int_repr(), qlast.int_repr()))
+        self.assertTrue(qr.q_scale() == qlast.q_scale())
+        self.assertTrue(qr.q_zero_point() == qlast.q_zero_point())
+        self.assertTrue(np.array_equal(qlast.dequantize().numpy(), qr.dequantize().numpy()))
```

Comment:
I think we have `self.assertEqual` on pytorch tensor as well. so
```
self.assertEqual(qlast.dequantize(), qr.dequantize())
```
should work

Reply:
ah, good point! when I tried it I typed assertEquals instead of assertEqual and we don't override that one :-P